universal_silabs_flasher/flasher.py

```diff
@@ -102,6 +103,20 @@ async def enter_yellow_bootloader(self):
             toggle_delay=0.1,
         )
 
+    async def enter_sonoff_bootloader(self):
+        _LOGGER.info("Triggering Sonoff bootloader")
+
+        baudrate = self._baudrates[ApplicationType.GECKO_BOOTLOADER][0]
+        async with connect_protocol(self._device, baudrate, SerialProtocol) as sonoff:
+            serial = sonoff._transport.serial
+            serial.dtr = False
+            serial.rts = True
+            time.sleep(0.1)
```

Comment:
Using `time.sleep` in an async context will lock up the event loop. Use `await asyncio.sleep` instead.

Reply:
fixed